pkg/generator/generator.go

```diff
@@ -273,17 +275,28 @@ func (c *GenVars) keySeparatorLookup(key, val string) string {
 		return val
 	}
 
-	pm := ParsedMap{}
+	keys, err := ajson.JSONPath([]byte(val), "$.."+kl[1])
+	if err != nil {
+		panic(err)
```

Comment:
instead of panic do an early return of empty and log.Debug or log.Error the error